scissors-sample/src/main/java/com/lyft/android/scissorssample/CropResultActivity.java

```diff
@@ -42,17 +43,17 @@ protected void onCreate(Bundle savedInstanceState) {
         String filePath = getIntent().getStringExtra(EXTRA_FILE_PATH);
         File imageFile = new File(filePath);
 
-        Picasso.with(this)
-                .load(imageFile)
-                .memoryPolicy(MemoryPolicy.NO_CACHE, MemoryPolicy.NO_STORE)
-                .into(resultView);
+//        Picasso.with(this)
```

Comment:
This is OK for testing, but try not to change too much outside of what is required, because we might want to try merging this upstream at some point, or merge the changes from the upstream repo down into our fork 😄

Reply:
I changed this while testing, since the app was calling Picasso instead of Glide. I can revert these changes.